src/Nethermind/Nethermind.Core/Json/EthereumJsonSerializer.cs

```diff
@@ -25,6 +25,11 @@ namespace Nethermind.Core.Json
 {
     public class EthereumJsonSerializer : IJsonSerializer
     {
+        public EthereumJsonSerializer()
+        {
+            _serializer = JsonSerializer.Create(_settings);
```

Comment:
We only instantiate one or two EthereumJsonSerializers inside the application. For testing, however, making it static would cause trouble.

This is actually a fix to a problem like the one you spotted - previously _serialize was created each time we were calling Deserialize method! :O